config/services.yaml

```diff
@@ -28,12 +28,12 @@ services:
         App\Application\Command\CommandHandlerInterface:
           public: true
           tags:
-              - { name: tactician.handler, typehints: true, bus: command }
+              - { name: messenger.message_handler, typehints: true, bus: messenger.bus.command }
```

Comment:
`typehints: true` should not be required

Reply:
Mmmm Ok. 

Btw CommandHandlerInterface and QueryHandlerInterface, should not they have an invoke method?

Reply:
Forget the question 🤣